Maple2.Model/Game/User/Mastery.cs

```diff
@@ -1,10 +1,14 @@
 ﻿using System.Runtime.InteropServices;
+using Maple2.PacketLib.Tools;
+using Maple2.Tools;
+using Maple2.Tools.Extensions;
 
 namespace Maple2.Model.Game; 
 
-[StructLayout(LayoutKind.Sequential, Pack = 4, Size = 48)]
-public struct Mastery {
+//[StructLayout(LayoutKind.Sequential, Pack = 4, Size = 48)]
+public class Mastery : IByteSerializable {
     private int Unknown;
```

Comment:
This was mostly needed as a placeholder for struct serialization. Since you are use `IByteSerializable` now, you can just have a `WriteInt();` for this.